framework/uicomponents/qml/Muse/UiComponents/menuitem.cpp

```
@@ -221,77 +265,159 @@ void MenuItem::setSubitems(const QList<MenuItem*>& subitems)
+    info.decoration.iconColor = Color::fromString(m_iconColor.toStdString());
+    info.decoration.checkable = m_checkable ? rcommand::Checkable::Yes : rcommand::Checkable::No;
+    return info;
+}
+
+void MenuItem::setCommand(const rcommand::Command& command)
+{
+    m_intent = command.toString();
+    setId(QString::fromStdString(m_intent));
+}
+
+muse::rcommand::Command MenuItem::command() const
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)